lib/ui/painting.dart

```diff
@@ -1340,6 +1347,18 @@ enum PathFillType {
   evenOdd,
 }
 
+/// Path operation enums
+/// 
+/// For use with [Path.combine]
+/// Must be kept in sync with SkPathOp
```

Comment:
This line should be a comment (`//`), not a dartdoc (`///`), so that it doesn't appear in the docs.